pom.xml

```diff
@@ -8,63 +8,55 @@
   <packaging>jar</packaging>
   <version>1.1.1</version>
   <name>ItemExchange</name>
-  <url>https://github.com/Civcraft/ItemExchange</url>
+  <url>https://github.com/DevotedMC/ItemExchange</url>
   
   <properties>
-        <maven.compiler.source>1.7</maven.compiler.source>
-        <maven.compiler.target>1.7</maven.compiler.target>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
   </properties>
   
-  <build>
-    <sourceDirectory>${basedir}/src</sourceDirectory>
-    
+  <build>    
     <resources>
       <resource>
         <directory>${basedir}</directory>
         <includes>
           <include>license.txt</include>
-        </includes>
-      </resource>
-      <resource>
-        <directory>${basedir}/src</directory>
-        <includes>
-          <include>**/*.java</include>
-          <include>*.yml</include>
-          <include>*.csv</include>
```

Comment:
Just tested here: https://build.devotedmc.com/job/Bastion-pullrequest/ws/target/classes/

answer is no :(